profiles/netcore2+.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <!-- All .NET Core / .NET Standard TFMs -->
+  <PropertyGroup>
+    <FakeItEasyTargetFrameworks>netstandard2.0;netstandard2.1</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>netstandard2.0;netstandard2.1</ValueTaskExtensionsTargetFrameworks>
+    <ExampleTargetFrameworks>netcoreapp3.0</ExampleTargetFrameworks>
+    <SpecsTargetFrameworks>netcoreapp2.1;netcoreapp3.0</SpecsTargetFrameworks>
+    <IntegrationTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.0</IntegrationTestsTargetFrameworks>
+    <UnitTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.0</UnitTestsTargetFrameworks>
+    <AnalyzerTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.0</AnalyzerTestsTargetFrameworks>
+    <TestHelpersTargetFrameworks>netstandard2.0;netstandard2.1</TestHelpersTargetFrameworks>
+    <ApprovalTestsTargetFrameworks>netcoreapp3.0</ApprovalTestsTargetFrameworks>
+  </PropertyGroup>
+</Project>

```

Comment:
Doesn't seem possible to install .NET Core 1.0 runtime on a recent Ubuntu, so I added a new profile to build just .NET Core 2+

Reply:
I can't even figure out how to install the runtime on my wsl!